themes/scss/widgets/themes/_light.scss

```diff
@@ -216,8 +216,20 @@ $chartMiniSelected: rgba(#333, 1);
     stroke-width: 2;
     fill: $commonBkg;
   }
+  .CDB-Chart-axisTipRect {
+    fill: #eee;
+  }
+  .CDB-Chart-axisTipRect--inverted {
+    font-weight: bold;
+    fill: $primary;
+  }
   .CDB-Chart-axisTipText {
-    fill: $link;
+    font-size: 11px;
```

Comment:
I probably did that while drafting the new tooltip style, I'll see what the class should be to avoid adding an extra font-size there.